privaterelay/settings.py

```diff
@@ -129,6 +130,9 @@
 
 # Application definition
 INSTALLED_APPS = [
+    'whitenoise.runserver_nostatic',
+    'django.contrib.staticfiles',
```

Comment:
Relevant for the back-enders: I've set up Whitenoise to also be able to serve the React app. See https://whitenoise.evans.io/en/stable/django.html#using-whitenoise-with-webpack-browserify-latest-js-thing (also linked below) and https://whitenoise.evans.io/en/stable/django.html#using-whitenoise-in-development.